src/content/en/updates/2017/06/supercharged-codesplit.md

```diff
@@ -0,0 +1,279 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: We implemented code splitting and route based chunking just like WebPack.
+
+{# wf_updated_on: 2017-06-22 #}
+{# wf_published_on: 2017-06-22 #}
+{# wf_tags: supercharged #}
+{# wf_featured_image: /web/updates/images/supercharged/poster.jpg #}
+{# wf_featured_snippet: We implemented code splitting and route based chunking just like WebPack. #}
```

Comment:
nit: this is the **hook** that shows up on /web/updates/ landing page. It should be more descriptive and engaging. If I don't know what code splitting is, or why I might want route based chunking, then I completely skip this.